### PR TITLE
Add README-Indonesian.md

### PR DESCRIPTION
**Pull request by andriannobella**
### Summary

Fixes #<NUM>

*Lorem ipsum dolor sit amet, consectetur adipiscing.*

### List of changes proposed in this PR (pull-request)

* *Lorem ipsum dolor sit amet, consectetur adipiscing.*
* *Lorem ipsum dolor sit amet, consectetur adipiscing.*

### What should a reviewer concentrate their feedback on?

- [ ] *Lorem ipsum dolor sit amet, consectetur adipiscing.*
- [ ] Everything looks ok?

### Acknowledging contributors

- [ ] All contributors to this pull request are already named in the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/master/README.md#contributors) in the README file.
- [x] The following people should be added to the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/master/README.md#contributors) in the README file:

### CODE REVIEW

**Comment by malvikasharan** [comment]
@andriannobella can you please suggest someone who can review in Indonesian? I am not sure if I know someone who I can call in to review this.

**Comment by fuadreza** [comment]
> @andriannobella can you please suggest someone who can review in Indonesian? I am not sure if I know someone who I can call in to review this.

I'm from Indonesia. I want to contribute to this issue..

File: README-Indonesian.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+
+_The Turing Way_ adalah sebuah panduan ringan untuk ilmu data yang dapat direproduksi.
```
Comment (fuadreza) [inline README-Indonesian.md:9]: ... panduan sederhana untuk **_data science_** yang ...

File: README-Indonesian.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+Anda dapat membacanya di sini: <https://the-turing-way.netlify.com>
+Anda saat ini sedang melihat repositori proyek GitHub tempat semua bit menyusun panduan dan tempat proses penulisan/pembuatan panduan.
```
Comment (fuadreza) [inline README-Indonesian.md:11]: ... tempat semua bit **yang membuat panduan ini tersedia** dan tempat proses penulisan/pembuatan panduan **terjadi** ...

File: README-Indonesian.md [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+
+Tujuan kami adalah menyediakan semua informasi yang dibutuhkan para peneliti di awal proyek mereka untuk memastikan bahwa mereka mudah untuk mereproduksi di akhir.
+
+Ini juga berarti memastikan mahasiswa PhD, postdoc, PI dan tim pendanaan mengetahui bagian mana dari "tanggung jawab reproduktifitas" yang dapat mereka pengaruhi, dan apa yang harus mereka lakukan untuk mendorong ilmu data menjadi lebih efisien, efektif, dan dapat dipahami.
```
Comment (fuadreza) [inline README-Indonesian.md:15]: ... dari "tanggung jawab **reproduktivitas**" yang dapat ... untuk mendorong **_data science_** menjadi lebih ...

File: README-Indonesian.md [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+
+Daftar Isi:
+
+- [Tentang proyek](#about-the-project)
+- [Tim](#the-team)
+- [Berkontribusi](#contributing)
+- [Mengutip _The Turing Way_](#citing-the-turing-way)
+- [Berhubungan](#get-in-touch)
```
Comment (fuadreza) [inline README-Indonesian.md:23]: **Kontak**

File: README-Indonesian.md [hunk 5]
```diff
@@ -0,0 +1,100 @@
...
+- [Kontributor](#contributors)
+
+🎧 Jika Anda lebih memilih audio untuk pengenalan proyek, anggota tim kami Rachael hadir di [Open Science Fair 2019](https://www.opensciencefair.eu/) di Porto dan demonya direkam di [Orion podcast](https://orionopenscience.podbean.com/e/the-fair-is-in-town-figshare-the-turing-way-and-open-science-quest-at-the-osfair2019/).
```
Comment (fuadreza) [inline README-Indonesian.md:26]: ... Rachel **yang hadir pada** ... **Penjelasan** _The Turing Way_ mulai pada menit ...

File: README-Indonesian.md [hunk 6]
```diff
@@ -0,0 +1,100 @@
...
+_The Turing Way_ ikhtisar mulai pada menit 5:13.
+
+### Tentang proyek
+
+Penelitian yang dapat direproduksi diperlukan untuk memastikan bahwa karya ilmiah dapat dipercaya.
+Pemberi dana dan penerbit mulai mewajibkan agar publikasi menyertakan akses ke data pokok dan kode analisis.
```
Comment (fuadreza) [inline README-Indonesian.md:32]: ... **analisis kode**.

File: README-Indonesian.md [hunk 7]
```diff
@@ -0,0 +1,100 @@
...
+Tujuannya adalah untuk memastikan bahwa semua hasil dapat diverifikasi dan dibangun secara independen dalam pekerjaan di masa mendatang.
+Ini terkadang lebih mudah diucapkan daripada dilakukan.
+Berbagi hasil penelitian ini berarti memahami manajemen data, ilmu perpustakaan, pengembangan perangkat lunak, dan teknik integrasi berkelanjutan: keterampilan yang tidak diajarkan atau diharapkan secara luas oleh para peneliti akademis dan ilmuwan data. Karena kegiatan ini tidak diajarkan secara umum, kami menyadari bahwa beban persyaratan dan perolehan keterampilan baru dapat mengintimidasi individu yang baru mengenal dunia ini.
```
Comment (fuadreza) [inline README-Indonesian.md:35]: ... para peneliti akademis dan **_data scientist_**

File: README-Indonesian.md [hunk 8]
```diff
@@ -0,0 +1,100 @@
...
+*The Turing Way* adalah sebuah buku pegangan untuk mendukung pelajar, pengawas pelajar, penyandang dana dan editor jurnal dalam memastikan bahwa ilmu data yang dapat direproduksi "terlalu mudah untuk tidak dilakukan" bahkan untuk orang yang belum pernah bekerja dengan cara ini sebelumnya.
```
Comment (fuadreza) [inline README-Indonesian.md:36]: ... memastikan bahwa **_data science_** yang dapat direproduksi ...

File: README-Indonesian.md [hunk 9]
```diff
@@ -0,0 +1,100 @@
...
+Ini akan mencakup materi pelatihan tentang kontrol versi, pengujian analisis, dan komunikasi terbuka dan transparan dengan para pengguna di masa mendatang, serta mengembangkan studi kasus dan lokakarya di Turing Institute.
```
Comment (fuadreza) [inline README-Indonesian.md:37]: ... pelatihan tentang **_version control_**, pengujian analisis ...

File: README-Indonesian.md [hunk 10]
```diff
@@ -0,0 +1,100 @@
...
+Proyek ini dikembangkan secara terbuka dan semua pertanyaan, komentar, dan rekomendasi diterima di repositori github kami: <https://github.com/alan-turing-institute/the-turing-way>.
+
+### Tim
+
+Ini adalah (bagian dari) tim pekerjaan perencanaan proyek di Turing Institute.
```
Comment (fuadreza) [inline README-Indonesian.md:42]: ...(bagian dari) **tim proyek yang sedang merencanakan pekerjaan** di Turing Institute.

File: README-Indonesian.md [hunk 11]
```diff
@@ -0,0 +1,100 @@
...
+Mereka menggambarkan berbagai konten dari buku pegangan, upaya kolaboratif dalam komunitas dan proyek _The Turing Way_ secara umum.
+Ilustrasi ini tersedia di Zenodo ([https://zenodo.org/record/3695300)](https://zenodo.org/record/3695300) di bawah lisensi CC-BY.
+
+Saat menggunakan salah satu gambar, harap sertakan atribusi berikut:
+> This image was created by Scriberia for The Turing Way community and is used under a CC-BY licence.
+
+Versi terbaru dari Zenodo dapat dikutip seperti:
+> The Turing Way Community, & Scriberia. (2020, March 3). Illustrations from the Turing Way book dashes. Zenodo. http://doi.org/10.5281/zenodo.3695300
+
+Kami telah menggunakan beberapa ilustrasi ini di [Welcome Bot](https://github.com/apps/welcome)'s tanggapan atas kontribusi anggota baru di repositori GitHub ini.
```
Comment (fuadreza) [inline README-Indonesian.md:82]: ... ilustrasi ini **pada tanggapan** [Welcome Bot](https://github.com/apps/welcome) **terhadap** kontribusi anggota baru ...

File: README-Indonesian.md [hunk 12]
```diff
@@ -0,0 +1,100 @@
...
+
+### Berhubungan
```
Comment (fuadreza) [inline README-Indonesian.md:84]: ### **Kontak**

**Review by fuadreza — CHANGES_REQUESTED** [review]

**Comment by malvikasharan** [comment]
Thank you @fuadreza. @andriannobella can you please integrate review points?

**Review by malvikasharan — DISMISSED** [review]

**Comment by malvikasharan** [comment]
@all-contributors please add @fuadreza for translation

**Comment by malvikasharan** [comment]
@fuadreza Thank you so much for the review and edits. Please let me know when you are ready to merge this. 🎊

**Comment by fuadreza** [comment]
> @fuadreza Thank you so much for the review and edits. Please let me know when you are ready to merge this. confetti_ball

You're welcome. what can I do? the Change request has not been made yet right?

**Comment by malvikasharan** [comment]
@andriannobella I see that the comments by @fuadreza have not been integrated (It is outdated for the branch but should be easy to integrate into edit mode on GitHub or locally). Can you please take some time to integrate changes? Let me know if I or @fuadreza can help.

**Comment by andriannobella** [comment]
> @andriannobella I see that the comments by @fuadreza have not been integrated (It is outdated for the branch but should be easy to integrate into edit mode on GitHub or locally). Can you please take some time to integrate changes? Let me know if I or @fuadreza can help.

Hi @malvikasharan Please tell me how to integrate it?

**Comment by malvikasharan** [comment]
@andriannobella You will have to manually integrate the changes proposed by @fuadreza by editing this file directly: https://github.com/andriannobella/the-turing-way/blob/indonesian/README-translated/README-Indonesian.md. I can't read Indonesian so it's a bit hard for me to do that. I hope you can do it. I will unresolved the suggestions that you marked as resolved but have not introduced in your original draft.

File: README-Indonesian.md (hunk 1 above)
Comment (malvikasharan) [reply README-Indonesian.md]: I integrated this, please double-check before resolving this.

File: README-Indonesian.md (hunk 2 above)
Comment (malvikasharan) [reply README-Indonesian.md]: Not fully integrated yet, please edit and integrate this.

**Comment by andriannobella** [comment]
Hi @malvikasharan I've already integrated the changes manually.
Please check it out.

**Review by malvikasharan — APPROVED** [review]
I am approving this. Thank you so much @andriannobella and @fuadreza. Please let me know when you are ok to merge this.

**Comment by andriannobella** [comment]
> I am approving this. Thank you so much @andriannobella and @fuadreza. Please let me know when you are ok to merge this.

Hi @malvikasharan I am ok to merge it

**Pull request merged**